src/org/omegat/core/data/RealProject.java

```diff
@@ -463,6 +465,39 @@ public Map<String, TMXEntry> align(final ProjectProperties props, final File tra
         return alignFilesCallback.data;
     }
 
+    /**
+     * Alternative Align project.
+     */
+    public Map<EntryKey, TMXEntry> alignAlt(final ProjectProperties props, final File translatedDir)
```

Comment:
It is better to consolidate function to single one with flag for behavior, because ``alignAlt`` and ``align`` share similar function.